src/scripts/Dockerfile.android

```diff
@@ -0,0 +1,22 @@
+FROM debian:stretch AS debian
+ARG SDK
+RUN apt-get update && apt-get -y --no-install-recommends install \
+	libboost-all-dev \
+	make \
+	python3-sphinx
+
+RUN mkdir -p /botan/android
+WORKDIR /botan
+COPY --from=bitriseio/android-ndk /opt/android-ndk android-ndk
+COPY configure.py configure.py
+COPY src src
+COPY doc doc
+COPY license.txt license.txt
+COPY news.rst news.rst
+ENV PATH=$PATH:android-ndk/toolchains/llvm/prebuilt/linux-x86_64/bin/
+RUN ./configure.py --prefix=android/arm --os=android --cpu=arm --cc=clang --cc-bin=armv7a-linux-androideabi${SDK}-clang++ --ar-command=arm-linux-androideabi-ar --with-sphinx
```

Comment:
For running this in CI might be best to take an additional ARG which specifies aarch64 vs armv7, then splitting it into two Travis builds.

Reply:
I would need in this case to handle clang binary name difference between those two arches too.